### DISCUSSION TITLE
Filtering Relationship: Column 'price' in where clause is ambiguous

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#441] [Q&A]
Data Source : `public function datasource(): ?Builder
    {
        return Order::query()
            ->join('links', function ($links) {
                $links->on('orders.link_id', '=', 'links.id');
            })
            ->select([
                'orders.*',
                'links.live_url as live_url',
            ]);
    }`

relationSearch : `public function relationSearch(): array
    {
        return [
            'link' => [
                'live_url', // i dont know what to put here to stop it from being ambiguous while fileting by price  :(
            ],
        ];
    }`

basically i have column price in both Order and Link model , having relationship link() in Order Modal, i did check the docs for filtering by relationship i dont think i really undrestood, what should i put in relationSearch function to filter only by Order's price

 
![Screenshot_1](https://user-images.githubusercontent.com/81111919/169274465-61730f54-2e36-4756-86fe-561e85110ec2.png)

**Comment by younus93** [comment]
Did you get this to work?
